docs/AudioSetup.md

```diff
@@ -65,10 +65,10 @@ ctl.!default {
 
 Let's break this down:
 
-* in `pmc` the `p` stands for playback. So were configuring playback settings inside this block
+* in `pcm` stands for r `Pulse Code Modulation`.  Here we configure the playback settings inside this block
```

Comment:
What this lonely `r` means? :D
also could this `Here we configure the playback settings inside this block` be `Inside this block we configure the playback settings`?